### PR TITLE
Add codenet model yaml and md files

### PR DESCRIPTION
**Pull request by kmh4321**
Signed-off-by: Karthik Muthuraman <karthik.muthuraman@ibm.com>

### CODE REVIEW

**Comment by kmh4321** [comment]
cc: @ckadner 

Fixes #38

File: model-samples/codenet-language-classification/codenet-language-classification.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+| ------------- | --------  | -------- |
+| This repository | [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0) | [LICENSE](LICENSE) |
+| Model Weights | [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0) | [LICENSE](LICENSE) |
+| Model Code (3rd party) | [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0) | [LICENSE](LICENSE) |
+| Test samples | [CDLA-Permissive 2.0](https://cdla.io) | [samples README](samples/README.md) |
+
+## Pre-requisites
+
+* `docker`: The [Docker](https://www.docker.com/) command-line interface. Follow the [installation instructions](https://docs.docker.com/install/) for your operating system.
+* The minimum recommended resources for this model are [SET NECESSARY GB] Memory and [SET NECESSARY CPUs] CPUs.
```
Comment (ckadner) [inline model-samples/codenet-language-classification/codenet-language-classification.md:29]: @kmh4321 do you know `[SET NECESSARY GB]` and `[SET NECESSARY CPUs]`?

**Review by ckadner — COMMENTED** [review]
I made a few updates, like removing references to Quay.io since the model is not there (yet). There are a few placeholders I do not know how to replace.

File: model-samples/codenet-language-classification/codenet-language-classification.md (hunk 1 above)
Comment (ckadner) [reply model-samples/codenet-language-classification/codenet-language-classification.md]: 2 GB 2 CPUs

File: model-samples/codenet-language-classification/codenet-language-classification.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+$ docker run -it -p 5000:5000 codenet-language-classifier
+```
+
+### 3. Use the Model
+
+The API server automatically generates an interactive Swagger documentation page. Go to `http://localhost:5000` to load it. From there you can explore the API and also create test requests.
+
+Use the `model/predict` endpoint to load a test file (you can use one of the test files from the `samples` folder) and get predicted language and probabilites from the API.
+
+![INSERT SWAGGER UI SCREENSHOT HERE](docs/swagger-screenshot.png)
```
Comment (ckadner) [inline model-samples/codenet-language-classification/codenet-language-classification.md:114]: @drewbutlerbb4 -- do you resolve the image references in the MLX UI code?

**Comment by ckadner** [comment]
@kmh4321 -- I thought we had decided to print the first 3 highest predictions, if there are more than one? When I run the prediction on a Python script I get 85% match for Haskel. It would be interesting to see the next best prediction.

**Curl**
```
curl -X POST "http://0.0.0.0:5000/model/predict" -H  "accept: application/json" -H  "Content-Type: multipart/form-data" -F 
"file=@animal_tasks.py;type=text/x-python-script"
```
**Request URL**
```
http://0.0.0.0:5000/model/predict
```
<h4>Server response</h4>

Code: 200

Response body:
```
{
  "status": "ok",
  "predictions": [
    {
      "language": "Haskell",
      "probability": 0.8541885018348694
    }
  ]
}
```

**Review by ckadner — APPROVED** [review]
/lgtm

Thanks Karthik!

**Pull request merged**